src/component/Header/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Header.css';
+
+const Header = () => (
+  <header className="header">
+    <div className="left-header">
+      <h1 className="logo">
+        <img src="../Header/image/logo.png" alt="logo" />
+      </h1>
+      <nav className="navigation">
+        <h2>Space Traveler`s Hub</h2>
+      </nav>
+    </div>
+    <div className="right-header">
+      <li>
+        <Link to="/">Rockets</Link>
+      </li>
+      <li>
+        <Link to="/mission">Mission</Link>
+      </li>
+      <span>|</span>
+      <li>
+        <Link to="/profile">MyProfile</Link>
```

Comment:
You did a great job building headers, I kindly suggest you to use `NavLink` instead of using `Link` because `NavLink` by default will add `active` class in a tag.

`import { NavLink } from react-router-dom`
`<NavLink to ='mission' exact="true"> Mission</NavLink>`

Reply:
Thanks for the review, I have implemented the changes based on your suggestions